fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/api/event/registry/v1/RegistryBlockEntityAddedCallback.java

```diff
@@ -0,0 +1,16 @@
+package net.fabricmc.fabric.api.event.registry.v1;
+
+import net.minecraft.block.entity.BlockEntity;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+public interface RegistryBlockEntityAddedCallback {
```

Comment:
we dont need a callback for every single type of registry

Reply:
it helps to have them

Reply:
could you please give a good example use case of these?

Reply:
iterating over all registry entries. 
if I wanted to make a modded entity list mod it is the easiest way to do it, because the entity map might still not contain all modded entities. same for block entities

Reply:
i guess thats a fair use case